Gemfile

```diff
@@ -63,6 +63,7 @@ gem 'twilio-ruby'
 gem 'two_factor_authentication'
 gem 'typhoeus'
 gem 'uglifier', '~> 3.2'
+gem 'user_agent_parser'
```

Comment:
Is there a reason for introducing a new dep here and not using device detector which we already have installed?

Reply:
device detector doesn't give you shortened version names.  I thought something like "Chrome 71 on Mac OS X 10" was more preferable to the user vs "Chrome 71.0.3578.98 on Mac OS X 10_13_6".  We do store the full user agent so we can get to it if we want it.  I could just grab the first number but I'm not sure if that will be correct in all instances.  Your thoughts?

Reply:
Sounds like there was a reason for introducing a new dep and not using device detector.